code/API_definitions/quality-on-demand.yaml

```diff
@@ -940,6 +977,29 @@ components:
             code: CONFLICT
             message: Conflict with an existing session for the same device.
 
+    Generic422:
```

Comment:
We probably have to add the new UNIDENTIFIABLE_DEVICE (still in PR in Commonalities)

Reply:
Right ... I just copied from artifacts/CAMARA_common.yaml here.

Reply:
Copied now the addition from https://github.com/camaraproject/Commonalities/pull/256/files